routes/users.js

```diff
@@ -18,21 +18,24 @@ const imageValidator = require('../validations/image');
 const authenticated = require('../middlewares/authenticated');
 const authOwnership = require('../middlewares/authOwnership');
 
+// router.use(express.json());
+
 // User list
 router.get('/users', authAdmin, userList);
+
 // User edit
-router.patch('/users/:id', userAuth, userValidation.signup, imageValidator, awsImageUploader, userEdit);
+router.patch('/users/:id', userAuth, upload('image'), imageValidator, awsImageUploader, userEdit);
+
 // User get data
 router.get('/auth/me', getData);
```

Comment:
Como hay **middlewares ya definidos** para verificar si el usuario ha iniciado sesion, seria bueno utilizarlos aqui al definir las rutas, si el usuario esta autenticado, la informacion de ese usuario (solo el `id`) se pasa al objeto request para que este disponible para los demas middlewares

```suggestion
router.get('/auth/me', authenticated, getData);
```
es decir que luego en `getData`, se puede recuperar el id del usuario desde el request (solo el `id` porque es lo que se guardo en el token):
```js
  getData: async (req, res) => {
    // const token = req.headers.aut... No es necesario
    // const { id } = await verify... No es necesario
    const { id } = req.user; // obtenemos el dato desde el request
    // ...
```